### PR TITLE
Css frameworks

### PR DESCRIPTION
**Pull request by AdrianMikk**
Requesting feedback before merging.

### CODE REVIEW

File: feed/feed.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
```
Comment (sulenchy) [inline feed/feed.html]: The file name should be `index.html` and not `feed.html`

File: src/scss/index.scss [hunk 2]
```diff
@@ -1 +1,22 @@
+@import "variables";
+@import "./header";
+@import "./login";
+@import "./feed";
+@import "./profile";
+@import "../../node_modules/bootstrap/scss/bootstrap";
 
+$small-screen: 768px;
```
Comment (sulenchy) [inline src/scss/index.scss]: You don't need these. Bootstrap handles breakpoints well

**Review by sulenchy — COMMENTED** [review]
@AdrianMikk I believe this is a work in progress. A few issues to address:

- The forms are not properly styled. You can see an example of bootstrap form at https://getbootstrap.com/docs/5.3/examples/sign-in/
- The navbar style needs to be improved. With bootstrap, this is easy. You see an example of navbar at https://getbootstrap.com/docs/5.3/examples/headers/


There are other issues stated above. Please have a look and address them accordingly

File: feed/feed.html (hunk 1 above)
Comment (AdrianMikk) [reply feed/feed.html]: Fixed

File: src/scss/index.scss (hunk 2 above)
Comment (AdrianMikk) [reply src/scss/index.scss]: Fixed

**Pull request merged**